app/routes/admin/tenants.py

```diff
@@ -256,14 +337,46 @@ async def get_wallet_auth_token(
     return response
 
 
-@router.put("/{wallet_id}", response_model=Tenant)
+@router.put("/{wallet_id}", response_model=Tenant, summary="Update Tenant by wallet ID")
 async def update_tenant(
     wallet_id: str,
     body: UpdateTenantRequest,
     group_id: Optional[str] = group_id_query,
     admin_auth: AcaPyAuthVerified = Depends(acapy_auth_tenant_admin),
 ) -> Tenant:
-    """Update tenant by id."""
+    """
+    Update tenant by id
+    ---
+
+    Use this endpoint to update a tenant by its wallet ID.
+
+    A Holder can't have their roles updated. Only Issuers and Verifiers can have their roles updated.
```

Comment:
To be clear: a holder cannot be _promoted_ to an issuer or verifier. Trying to promote a holder will return a 409. Only issuers or verifiers can be promoted to both roles. This endpoint can't be used to revoke roles from an issuer or verifier.
^ let's try paraphrase that ... with GPT's help :-)

Can also note that image_url for issuers/verifiers will be updated on trust registry